### PR TITLE
Minor changes

### LINKED ISSUE #106: Handle NaN values in PyTorch dataloader & add warning message

### PR DESCRIPTION
**Pull request by 14thibea**
Fix of several issues:
- fix label code generation
- deal with multi-class metrics
- remove nan values in CapsDataset (closes #106)
- remove all occurrences of data group in MAPS when overwriting

It is also now possible to predict with a different label and label code than the one used for training (choice of the label code possible in API mode only).

### CODE REVIEW

**Comment by mdiazmel** [comment]
Hi @14thibea ,

Can you please rebase `dev` to see if the tests run without problems?

**Comment by mdiazmel** [comment]
LGTM, I'll merge!
